src/arg_desc.jl

```diff
@@ -2,6 +2,9 @@ const AStr = AbstractString
 const ColorType = Union{Symbol,Colorant,PlotUtils.ColorSchemes.ColorScheme,Integer}
 const TicksType = Union{AVec{Real},Tuple{AVec{Real},AVec{AStr}},Symbol}
 
+link_histogram   = "[`histogram`](https://docs.juliaplots.org/stable/api/#Plots.histogram-Tuple)"
```

Comment:
I think this is a good idea, although this is not really aimed toward being maintainable.

Reply:
an option would be to insert some sort of tag like e.g. `[:histogram:]` and then replace that with the link when building the ducumentation.

Ideally that would then end up in the docstring as well, but I don't see a way to make that happen